### PR TITLE
Fixes Bug ankidroid#8192.

### PR DESCRIPTION
**Pull request by imaryandokania**
## Purpose / Description 
The problem was with progressDialog title which was having space 

## Fixes
Fixes https://github.com/ankidroid/Anki-Android/issues/8192
## Approach
_How does this change address the problem?_
by changing "" to null 

## How Has This Been Tested?

Before fix

Screenshot 2021-03-14 at 9 09 53 AM

Solved by testing in emulator(After Fix)

Screenshot 2021-03-14 at 9 08 33 AM

Tested on emulator: Pixel 3a API R

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/MyAccount.java [hunk 1]
```diff
@@ -233,7 +233,7 @@ public void onProgressUpdate(Object... values) {
         public void onPreExecute() {
             Timber.d("loginListener.onPreExecute()");
             if (mProgressDialog == null || !mProgressDialog.isShowing()) {
-                mProgressDialog = StyledProgressDialog.show(MyAccount.this, "",
+                mProgressDialog = StyledProgressDialog.show(MyAccount.this,null,
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/MyAccount.java:236]:
```suggestion
                mProgressDialog = StyledProgressDialog.show(MyAccount.this, null,
```

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
nitpick of the century... other than that, good to go!

**Review by david-allison-1 — APPROVED** [review]
Thanks for the patience. Let's do this. 🚀🚀🚀🚀🚀

Awaiting CI to pass

**Comment by david-allison-1** [comment]
Thanks so much!

**Comment by Arthur-Milchior** [comment]
I personally would have asked for more descriptive commit title, to help future browsing of commit list aand git blame. Thanks for noticing and fixing

**Comment by imaryandokania** [comment]
> I personally would have asked for more descriptive commit title, to help future browsing of commit list aand git blame. Thanks for noticing and fixing

Definitely i will take care of these things in my future contributions !

**Pull request merged**